### PR TITLE
Fix PHP5.4.40 parsing error

### PR DESCRIPTION
**Pull request by doekia**
Fix PHP5.4.40 parsing error

### CODE REVIEW

File: ACL/AclXmlFileloader.php [hunk 1]
```diff
@@ -69,7 +69,8 @@ public function load(Module $module)
     protected function parseAcls(SimpleXMLElement $xml, Module $module)
     {
         //If there are no acl node continue to parse the xml
-        if (false === $acls = $xml->xpath('//config:acls/config:acl')) {
+        $acls = $xml->xpath('//config:acls/config:acl');
```
Comment (lovenunu) [inline ACL/AclXmlFileloader.php:72]: Well, this one doesn't matter with php 5.4 :)

**Comment by lovenunu** [comment]
Thanks :)

File: ACL/AclXmlFileloader.php (hunk 1 above)
Comment (doekia) [inline ACL/AclXmlFileloader.php:72]: You are right. It was more some kind of standardisation - feel free to dismiss it

**Comment by bcbrr** [comment]
Thanks for the fix !
I'll leave the first change for consistency.
Would you like to be added in the CONTRIBUTORS file ? If so, under what name ?

**Comment by doekia** [comment]
For such minimalist change, don't bother with the contributors file .... :-)

**Pull request merged**